data/datasets/oa_camel/README.md

```diff
@@ -0,0 +1,23 @@
+Dataset Description
+This is the chemistry, biology, math, and physics datasets created by CAMEL ai. https://huggingface.co/camel-ai They have been combined and converted to the Open Assistant format.
+
+There are 110000 entries.
+
+Languages
+English
+
+Dataset Structure
+This dataset follows the OA format, which is:
+
+INSTRUCTION (string): Instruction text
+RESPONSE (string): Expected response to the instruction
+SOURCE (string): Original data source short name, e.g. "wikipedia"
```

Comment:
Please describe the contents of the columns for CamelAI instead (like the keys and values for METADATA)